Description: Fix libdir location
 Everything assumes that libs are going to be in the correct $host folder,
 but it was changed in d9681c3747ff4f to override it to /usr/lib64, which
 then breaks packaging of things that are trying to glob lib64/*/libspandsp*
 when there's nothing to match the *

--- spandsp-3.0.0.orig/configure.ac
+++ spandsp-3.0.0/configure.ac
@@ -270,7 +270,7 @@ then
     case "${host}" in
     x86_64-*)
         # X86_64 Linux machines may have both 64 bit and 32 bit libraries. We need to choose the right set
-        AX_CHECK_REAL_FILE([${prefix}/lib64], libdir='${exec_prefix}/lib64')
+        AX_CHECK_REAL_FILE([${prefix}/lib64/${host}], libdir='${exec_prefix}/lib64/${host}')
         AX_CHECK_REAL_FILE([/usr/X11R6/lib64], [TESTLIBS="$TESTLIBS -L/usr/X11R6/lib64"], AC_CHECK_FILE([/usr/X11R6/lib], [TESTLIBS="$TESTLIBS -L/usr/X11R6/lib"]))
         # The very oldest AMD 64 bit chips support SSE2, SSE and MMX
         enable_sse2="yes"
